app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find(params[:followed_id])
+    current_user.follow(@user)
+    respond_to do |format|
+      format.html {redirect_to @user}
+      format.js
+    end
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
why you have `show` action in `relationship_controller` but you still keep two action `show` in both `followings_controler` and `followers_controller` ???
dont copy if you do not understand it :-1:

Reply:
:v em quên chưa xóa "only". :v